### ISSUE TITLE
No working on 0.84.0b4

### ISSUE THREAD

**Issue by pepeEL** [custom-components/custom_updater#64]
After update to beta home assistant 0.84.0b4 custom updater not working. No show custom card only show version custom updater.

**Comment by ludeeus** [comment]
Do **not** remove the issue template, without you supplying all the info that is requested in it, I have no idea on what the problem might be...

**Comment by pepeEL** [comment]
Please test it on latest beta HA 0.84.0b4
It look as on image:
![custom](https://user-images.githubusercontent.com/16692223/49806124-318b6080-fd57-11e8-9ba8-7a0ad4e884fc.jpg)

**Comment by ludeeus** [comment]
Sure... But you still need to fill out the issue template with configuration and debug logs..

**Comment by bieniu** [comment]
I think the issue occurs if the lovelace configuration is in HA storage and the ui-lovelace.yaml file is removed.

**Comment by ludeeus** [comment]
That will be an issue yes, only ui-lovelace.yaml can be supported, will try to test that tomorrow (UTC)

**Comment by pepeEL** [comment]
i have configuration in storage but no delete old file ui-lovelace.yaml

**Comment by pepeEL** [comment]
today was relesed new official 0.84.1 Home Assistant

**Comment by ludeeus** [comment]
Just tested this and it does still work with Home Assistant 0.84

**Comment by pepeEL** [comment]
Yes in this mode works ok. But please test it with store configuration in /. Storage

**Comment by Ernst79** [comment]
Same issue here, doesn't work anymore after switching to storage mode.

**Comment by ludeeus** [comment]
I will **not** make it work with .sorrage.
**only** Home Assistant should write to the files there.
